raiden/resolver/client.py

```diff
@@ -0,0 +1,63 @@
+import json
```

Comment:
The main function in this file is `reveal_secret_with_resolver`. This function is call from `raiden_event_handler.py` by `handle_send_secretrequest`:

- once the payee got a lockedpayment before sending a `secretrequeste` to the payer, it will call `reveal_secret_with_resolver` to try to find the secret in a configured resolver.

As such, I'm not moving the file to /tools.

I rearranged it a bit so it will not look like "testing code"